### PR TITLE
TF-558: Web quick start

### PR DESCRIPTION
**Pull request by JordanBerni-UL**
- [ ] Test plan updated (by PR creator) - Speak to Lesley once #308 is merged as well. Covered in https://ultrahaptics.atlassian.net/browse/TF-591
- [x] Code Review
- [x] Developer Testing: Key item tests
- [x] Product Owner Review (Optional)
- [x] Update Release Notes on the JIRA ticket for harvest at release time

Video of workflow:
https://user-images.githubusercontent.com/107030829/177536179-0bdf18ce-2059-455c-ad1f-075363fc599b.mp4

### CODE REVIEW

File: TF_Settings_Web/package.json [hunk 1]
```diff
@@ -12,12 +12,15 @@
...
+        "react-canvas-confetti": "^1.3.0",
+        "react-circular-progressbar": "^2.1.0",
         "react-dom": "^17.0.2",
         "react-scripts": "5.0.0",
+        "react-transition-group": "^4.4.2",
         "typescript": "^4.5.5",
         "web-vitals": "^2.1.4"
     },
-    "homepage": "/settings/",
+    "homepage": "/camera",
```
Comment (GabrielHomewoodUL) [inline TF_Settings_Web/package.json]: I think we will need to confirm this works with how we are serving the settings from the .net service

File: TF_Settings_Web/src/Components/Controls/IconTextButton.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    text,
+    textStyle,
+    onClick,
+}) => {
+    const [hovered, setHovered] = React.useState<boolean>(false);
+    const [pressed, setPressed] = React.useState<boolean>(false);
+
+    return (
+        <button
+            className={`IconTextButton ${hovered ? ' IconTextButtonHovered' : ''} ${
```
Comment (GabrielHomewoodUL) [inline TF_Settings_Web/src/Components/Controls/IconTextButton.tsx]: I don't think you need the spaces before the classes as they are already between the `${}` sections

File: TF_Settings_Web/src/Components/Pages/Camera/CalibrationComponents.tsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+import IconTextButton from '../../Controls/IconTextButton';
+
+interface CalibrateInstructionsProps {
+    progress: number;
+    containerStyle: CSSProperties;
+}
+export const CalibrateInstructions: React.FC<CalibrateInstructionsProps> = ({ progress, containerStyle }) => {
+    const instructionsText = (
+        <h1>
+            Hold INDEX FINGER against <br /> this <span style={{ color: '#01EB85' }}>GREEN CIRCLE</span>
```
Comment (GabrielHomewoodUL) [inline TF_Settings_Web/src/Components/Pages/Camera/CalibrationComponents.tsx]: Could we move the colour into a CSS class? It looks like it's being used in a few places and that way we could update it everywhere easily

File: TF_Settings_Web/src/Components/Pages/Camera/CameraPage.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            <Route
+                path="quick"
+                element={
+                    <CameraPosition
+                        activePosition={activePosition}
+                        setPosition={(position: PositionType) => setActivePosition(position)}
+                    />
+                }
+            />
+            <Route path="quick/calibrate/*" element={<CameraCalibratePage activePosition={activePosition} />} />
```
Comment (GabrielHomewoodUL) [inline TF_Settings_Web/src/Components/Pages/Camera/CameraPage.tsx]: What's the purpose of the `/*` at the end of the path?

Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/Pages/Camera/CameraPage.tsx]: That's to let the router know that the 'quick/calibrate' route has children (e.g. quick/calibrate/top)

File: TF_Settings_Web/src/Components/Controls/IconTextButton.tsx (hunk 2 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/Controls/IconTextButton.tsx]: Good catch 👍

**Review by GabrielHomewoodUL — APPROVED** [review]

**Pull request merged**